src/components/input-dropdown/input-dropdown.component.html

```diff
@@ -32,10 +34,11 @@
                     class="form-control"
                     [(ngModel)]="filterText"
                     (input)="filterChange.emit(filterText)"
-                    (click)="$event.stopPropagation()" />
+                    (click)="$event.stopPropagation()"
+                    aria-label="Filter"/>
```

Comment:
This would prevent the aria label from being localized. Might I recommend the approach Angular Material uses for their [select](https://github.com/angular/components/blob/master/src/material/select/select.ts#L421) component adding an input on the component, eg:

```typescript
 /** Aria label of the input. If not specified, the placeholder will be used as label. */
@Input('aria-label') ariaLabel: string = '';
```

And by default in the HTML use the placeholder if no aria label has been specified:

```html
[attr.aria-label]="ariaLabel || placeholder"
```

Reply:
Very good. I changed it.